styles/lazy-styles.css

```diff
@@ -97,6 +97,10 @@
   transition: all .3s ease-in-out;
 }
 
+.modal-container img {
```

Comment:
This rule applies to all images in all modals. Can we make it more specific? Like adding `.thank-you-for-downloading img`